lib/elixir_learn_phoenix/product.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ElixirLearnPhoenix.Product do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "products" do
+    field :name, :string
+    field :price, :integer
+    field :sold, :integer
+    field :rating, :float
```

Comment:
rating ko dùng để tính toán thì dùng `text` đc r, tránh những convert ko cần thiết